### PR TITLE
Add REST API notice

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Add a notice about the REST API that will cause this
project to be deprecated with Podman v2.0.

Fixes: #80

Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
@mheon PTAL

**Comment by mheon** [comment]
The Varlink API is already deprecated and in maintenance mode. We're not adding new features.

File: README.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ Python podman library.
 
 Provide a stable API to call into.
 
+**Notice:** When Podman version 2.0 is released in 2020, it will include a new REST API that will replace the varlink interface.  At the time of the release of Podman v2.0, the varlink interface will be deprecated, and at a later release removed from Podman.  Starting with Podman v1.9.0, the REST API is available for test usage.  The documentation for the REST API resides [here](http://docs.podman.io/en/latest/_static/api.html#operation/changesContainer).
```
Comment (mheon) [inline README.md:18]: It's actually the v1.8.0 release where `podman system service` became available, albeit in alpha form.

**Comment by TomSweeneyRedHat** [comment]
@mheon, thx, touched up and pushed updated.

**Comment by rhatdan** [comment]
LGTM

**Comment by TomSweeneyRedHat** [comment]
I've addressed @mheon's comments and have a LGTM from @rhatdan, can this be merged?  We just got a question about this change this morning.

**Comment by mheon** [comment]
LGTM

**Pull request merged**